src/cloud-api-adaptor/entrypoint.sh

```diff
@@ -94,11 +94,11 @@ azure() {
 gcp() {
     test_vars GCP_CREDENTIALS GCP_PROJECT_ID GCP_ZONE PODVM_IMAGE_NAME
 
-    [[ "${PODVM_IMAGE_NAME}" ]] && optionals+="-gcp-image-name ${PODVM_IMAGE_NAME} "
+    [[ "${PODVM_IMAGE_NAME}" ]] && optionals+="-imageid ${PODVM_IMAGE_NAME} "
     [[ "${GCP_PROJECT_ID}" ]] && optionals+="-gcp-project-id ${GCP_PROJECT_ID} "
     [[ "${GCP_ZONE}" ]] && optionals+="-gcp-zone ${GCP_ZONE} "                         # if not set retrieved from IMDS
-    [[ "${GCP_MACHINE_TYPE}" ]] && optionals+="-gcp-machine-type ${GCP_MACHINE_TYPE} " # default e2-medium
-    [[ "${GCP_NETWORK}" ]] && optionals+="-gcp-network ${GCP_NETWORK} "                # defaults to 'default'
+    [[ "${GCP_MACHINE_TYPE}" ]] && optionals+="-instance-type ${GCP_MACHINE_TYPE} "    # default e2-medium
```

Comment:
Personally I think it's more intuitive to keep the options aligned to the cloud provider nuances to the extent possible. Like instance-type (aws), instance-size( azure).  
I see all public GCP docs refers to machine type (for cloud instances) so may be it's ok to use `machine-type` for GCP. Wdyt ?

Reply:
I mostly agree with you, but I have mixed feelings. I like the idea of using the provider’s naming internally, but for the user interface, I think it makes sense to go with more generic names when possible (which we don’t really do today). That way, if users switch providers, it’s less disruptive for them.

I haven’t thought too much about it, so maybe it wouldn’t work for every variable because of specific nuances. But in general, it’d help keep the interface more consistent across providers.

For now, I’m good with keeping machine-type.